qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -200,47 +186,50 @@ def result(
         """Return the result of the job.
 
         Note:
-            Some IBMQ job results can be read only once. A second attempt to
-            query the API for the job will fail, as the job is "consumed".
-
-            The first call to this method in an ``IBMQJob`` instance will query
-            the API and consume the job if it finished successfully (otherwise
-            it will raise a ``JobError`` exception without consuming the job).
-            Subsequent calls to that instance's method will also return the
-            results, since they are cached. However, attempting to retrieve the
-            results again in another instance or session might fail due to the
-            job having been consumed.
-
-            When `partial=True`, the result method returns a `Result` object
-            containing partial results. If partial results are returned, precaution
-            should be taken when accessing individual experiments, as doing so might
-            cause an exception. Verifying whether some experiments of a job failed can
-            be done by checking the boolean attribute `Result.success`.
-
-            For example:
-                If there is a job with two experiments (where one fails), getting
-                the counts of the unsuccessful experiment would raise an exception
-                since there are no counts to return for it:
-                i.e.
-                    try:
-                        counts = result.get_counts("failed_experiment")
-                    except QiskitError:
-                        print("Experiment failed!")
+            Some IBM Quantum Experience job results can be read only once. A
+            second attempt to query the server for the same job will fail, as the
+            job is "consumed".
```

Comment:
Would it sound better to say, `...will fail, since the job has already been "consumed"`?